packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -0,0 +1,50 @@
+import { IForwarder, IPrismInput } from '@stoplight/prism-core';
+import { IHttpOperation, IServer } from '@stoplight/types';
+import axios from 'axios';
+import { IHttpConfig, IHttpRequest, IHttpResponse } from '../types';
+
+export class HttpForwarder
+  implements IForwarder<IHttpOperation, IHttpRequest, IHttpConfig, IHttpResponse> {
+  public async forward(opts: {
+    resource?: IHttpOperation;
+    // todo: IHttpRequest should be enough
+    input: IPrismInput<IHttpRequest>;
+  }): Promise<IHttpResponse> {
+    if (!opts.resource) {
+      throw new Error('Missing spec');
+    }
+    if (!opts.resource.servers || opts.resource.servers.length === 0) {
+      throw new Error('Server list is missing in spec');
+    }
+
+    const response = await axios({
+      method: opts.input.data.method,
```

Comment:
Minor stylistic comment: how about moving that `opts.input.data` to a constant? You'd avoid the repetition.